plugins/modules/uptime_check.py

```diff
@@ -21,31 +21,59 @@
     name:
         required: true
         description:
-            - Name of the check
+            - A name must be given to identify the uptime check as a string. The name does not have to be unique.
     protocol:
         required: true
         description:
-            - The protocol used for the check, eg http, ping etc
+            - The type of check taking place must be specified as a string (e.g. http, tcp, ping).
     tags:
-        required: true
+        required: false
         description:
-            - The tag(s) to add to the check separated with ,
+            - Tags can be added to an uptime check to make them more organized and discoverable in the user interface. This attribute takes an array of strings where each tag must have a maximum length of 64 characters.
     timing:
-        required: true
+        required: false
+        description:
+            - The user can specify the number of minutes between each check. This attribute takes an integer, but defaults to 5 if not specified.
+    port:
+        required: false
+        description:
+            - A specific port number can be targetted on the destination URL by setting setting the port number as an integer. 
+    encryption:
+        required: false
+        description:
+            - The user can specify whether the uptime check uses encryption. This attribute takes a boolean (True or False), but defaults to False if not specified.
+    verify_certificate:
+        required: false
         description:
-            - The timing between the check running in minutes
+            - An uptime check can treat the target site as down if it has an invalid or unverifiable certificate if the boolean verify_certificate attribute is set to True. If not specified, this attribute defaults to False.
+    probe_filters:
+        required: false
+        description:
+            - The user can specify filters used for probe selection as an array of strings. Currently only region is supported (e.g. region:EU)
+    shouldcontain:
+        required: false
+        description:
+            - The uptime check will only determine that the target site is up if it contains a specified string.
+    integrationids:
+        required: false
+        description:
+            - The user can connect integrations which have been set up in the UI to the uptime check by specifying the integration IDs as a list of integers.
+    url:
+        required: false
+        description:
+            - A path on the destination server can be set for the uptime check to target. This is taken as a string.
     pause:
         required: false
         description:
-            - Not Required. Please set to "y" to pause the check on creation for testing
+            - To pause the uptime check on creation, set the value to "y". If left blank, the uptime check will begin running on creation.

```

Comment:
I think "y" and blank are unusual values. Something like this should really be a bool and accept True or False, with a default of False. 

We should also add types onto fields, there is a specification for what can be defined in the DOCUMENTATION string at https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#documentation-fields

Reply:
Removed the code to accept "y" so pause now works as a boolean.